auth/src/main/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivity.java

```diff
@@ -14,8 +14,10 @@
 
 package com.firebase.ui.auth.ui.idp;
 
+import android.annotation.SuppressLint;
```

Comment:
There are no changes in this file anymore besides these two imports and some whitespace, so maybe we can revert the whole file?